### ISSUE TITLE
Q: What to expect in the GPSR

### ISSUE THREAD

**Issue by MatthijsBurgh** [RoboCupAtHome/RuleBook#782]
## Summary
In the past the GPSR command generator of @kyordhel has been used. This is still mentioned in the rulebook in section 5.2. But the command generator has not been updated since 2019.

So my question is: Is the command generator still going to be used? Yes, in the current state or is it getting updated? No, please elaborate on what to expect.

P.S. I think the GPSR should be limited to skills required to solve any other stage 1 challenge.

**Comment by kyordhel** [comment]
For what I discussed with the TC, it is going to be used.

> We did not have time yet to extensively look at the gpsr generator. Right now we use the previous version.

I also asked the TC whether I should update the generation rules to be consistent with the latest rulebook and the reply I got was

> I think this challenge is confusing to a lot of newer teams. In practice it boasts a wide variety of skills that can be called upon but the rules state that any challenge from stage 1 or 2 may be called. This is way too difficult for the robots. As it introduces many potential points of failure. I think it would be best to stick to common skills the robots should have and focus this challenge on task planning based on a speech command.

I don't disagree with the general idea, but the EC vetoed these changes. The GPSR must grow as new skills are incorporated, not being reduced. Of course GPSR will not include tasks from Stage II tests from the past 3 years at the very least, but all challenges that once were part of State II that now considered as solved are safe to be requested to robots. EEGPSR should include everything that is being tested in the competition.

My suggestion was to split further [EE]GPSR in categories so teams have control on what the robot can do. It is pointless to include manipulation in SSPL but all robots should be capable of taking out the garbage even though the test was dropped.

As soon as I have a final version on which I can work on, I can update the generator grammars.
